build/WinObjC.Frameworks/WinObjC.Frameworks.nuproj

```diff
@@ -90,7 +96,6 @@
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\MediaPlayer\dll\MediaPlayer.vcxproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\MessageUI\dll\MessageUI.vcxproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\Metal\dll\Metal.vcxproj" />
-    <ProjectReference Include="$(MSBuildThisFileDirectory)..\MobileCoreServices\dll\MobileCoreServices.vcxproj" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\OpenGLES\dll\OpenGLES.vcxproj" />
```

Comment:
(It does have Core in the name for what its worth)

---
In reply to: [102297109](https://github.com/Microsoft/WinObjC/pull/1971#discussion_r102297109) [](ancestors = 102297109,102296613)